src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceCodeAction.cs

```diff
@@ -39,19 +42,45 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
 
                 if (moveToNamespaceResult.Succeeded)
                 {
-                    return SpecializedCollections.SingletonEnumerable(new ApplyChangesOperation(moveToNamespaceResult.UpdatedSolution));
+                    return CreateRenameOperations(moveToNamespaceResult);
                 }
             }
 
             return SpecializedCollections.EmptyEnumerable<CodeActionOperation>();
         }
 
+        private static ImmutableArray<CodeActionOperation> CreateRenameOperations(MoveToNamespaceResult moveToNamespaceResult)
+        {
+            Debug.Assert(moveToNamespaceResult.Succeeded);
+
+            var operations = PooledObjects.ArrayBuilder<CodeActionOperation>.GetInstance();
+            operations.Add(new ApplyChangesOperation(moveToNamespaceResult.UpdatedSolution));
+
+            var symbolRenameCodeActionOperationFactory = moveToNamespaceResult.UpdatedSolution.Workspace.Services.GetService<ISymbolRenamedCodeActionOperationFactoryWorkspaceService>();
+
+            // It's possible we're not in a host context providing this service, in which case
+            // just provide a code action that won't notify of the symbol rename
```

Comment:
Expand this comment to explain how the `SymbolRenamedOperation` is used so that we understand what we lose when not creating these operations.